examples/bin/verify-java

```diff
@@ -19,6 +19,7 @@
 
 use strict;
 use warnings;
+use File::Basename;
 
 sub fail_check {
   my ($current_version) = @_;
```

Comment:
Need to update error message here to indicate that you can ${DRUID_JAVA_HOME} ${JAVA_HOME} or java installed on ${PATH}